draft-ietf-quic-qlog-main-schema.md

```diff
@@ -209,6 +210,39 @@ The main tenets for the qlog schema design are:
   group_ids can be used to tag events to a particular context)
 * Metadata is stored together with event data
 
+This is achieved by a logical logging hierarchy of:
+
+* Log file
+  * Trace(s)
+    * Event(s)
+
+Two log file schema are defined: QLogFile ({{qlog-file-schema}}) and QLogFileSeq
```

Comment:
I don't particularly like the "file schema" vs "event schema" as we currently have it... I understand WHY you're doing it that way, but it seems a bit incongruous/inconsistent (e.g., we don't explain how/why to define new FILE schema, nor have we ways to extend existing FILE schema, in contrast to EVENT schema). 

maybe this is just me coming from the "old" way of using "schema", but it doesn't feel completely right semantically to me atm.

Reply:
Discussed in editors' meeting. 

Make explicit in new `file_schema` field with its own URI namespace and IANA registration. Allows us to also get rid of `qlog_version` (implicit in file_schema now)

Rename `event_categories` to `event_schemas` maybe for consistency with `file_shema`

```
urn:ietf:params:qlog:file:
urn:ietf:params:qlog:events:

file_schema: "urn:ietf:params:qlog:file:sequential",
event_schemas: [
    "urn:ietf:params:qlog:events:rick#roll",
    "urn:ietf:params:qlog:events:rick#astley",
    "urn:ietf:params:qlog:events:rick#moranis",
    "urn:ietf:params:qlog:events:john#doe",
    "urn:ietf:params:qlog:events:john#candy",
    "urn:ietf:params:qlog:events:john#goodman",
    "https://example.com/032024/pickle.html#pepper",
    "https://example.com/032024/pickle.html#lilly",
    "https://example.com/032024/pickle.html#rick",
]
```